### ISSUE TITLE
Compilation fails due to old data_serializer version

### ISSUE THREAD

**Issue by Vera-Spoettl** [gmpassos/statistics#10]
Error caused by data_serializer-1.1.0

../../../../../.pub-cache/hosted/pub.dev/data_serializer-1.1.0/lib/src/extension.dart:528:37: Error: The method 'UnmodifiableUint8ListView' isn't defined for the class 'Uint8List'.
 - 'Uint8List' is from 'dart:typed_data'.
Try correcting the name to the name of an existing method, or defining a method named 'UnmodifiableUint8ListView'.
  Uint8List copyAsUnmodifiable() => UnmodifiableUint8ListView(copy());
                                    ^^^^^^^^^^^^^^^^^^^^^^^^^

Seems as if an upgrade to 1.2.0 is necessary.

**Comment by gmpassos** [comment]
Thx for the report.

Resolved and published:

`statistics: ^1.1.2`

**Comment by adriank** [comment]
Flutter depends on collection 1.18.0, so the fix doesn't work.

```Because statistics >=1.1.2 depends on data_serializer ^1.2.0 which depends on collection ^1.19.0, statistics >=1.1.2 requires collection ^1.19.0.
And because every version of flutter from sdk depends on collection 1.18.0, statistics >=1.1.2 is incompatible with flutter from sdk.
So, because fluidpm depends on both flutter from sdk and statistics ^1.1.2, version solving failed.
```

**Comment by gmpassos** [comment]
> Flutter depends on collection 1.18.0, so the fix doesn't work.
> 

I have published `data_serializer: ^1.2.1`. Try it and let me know:

```text
 1.2.1

- Rollback to `collection 1.18.0` for Flutter SDK compatibility.

- collection: ^1.18.0

```
